### PR TITLE
WFNEWS-2110 - Evac Order and components

### PR DESCRIPTION
**Pull request by sukhpalp**

### CODE REVIEW

File: client/wfnews-war/src/main/angular/src/app/components/public-event-page/evac-order-details/evac-order-details.component.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    label: 'Evacuation Information',
+    iconPath: '/assets/images/svg-icons/link.svg',
+    componentStyle: {
+      backgroundColor: '#B91D38',
+      labelColor: '#FFFFFF',
+      iconColor: '#FFFFFF',
+      border: 'none'
+    },
+    clickHandler: () => {
+      window.open('https://www.emergencyinfobc.gov.bc.ca/');
```
Comment (yzlucas) [inline client/wfnews-war/src/main/angular/src/app/components/public-event-page/evac-order-details/evac-order-details.component.ts:34]: can we do window.open(url, '_blank') to open a new tab?

File: client/wfnews-war/src/main/angular/src/app/app.module.ts [hunk 2]
```diff
@@ -44,10 +45,16 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { RouteReuseStrategy } from '@angular/router';
 import { ServiceWorkerModule } from '@angular/service-worker';
 import { YouTubePlayerModule } from '@angular/youtube-player';
+import { NotificationSnackbarComponent } from '@app/components/notification-snackbar/notification-snackbar.component';
 import { DialogExitComponent } from '@app/components/report-of-fire/dialog-exit/dialog-exit.component';
 import { RoFReviewPage } from '@app/components/report-of-fire/review-page/rof-review-page.component';
+import { notificationMapComponent } from '@app/components/saved/add-saved-location/notification-map/notification-map.component';
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/app.module.ts:51]: I know you're not introducing this here but I think this component name should be capitialised so there is uniformity with the rest of the components in the project.

File: client/wfnews-war/src/main/angular/src/app/components/common/advisory-section/advisory-section.component.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+          font-size: 16px;
+          line-height: 24px;
+          font-weight: 400;
+          color: #484848;
+          font-family: "Noto sans", "BCSans", "Open Sans", Verdana, Arial, sans-serif;
+        }
+      }
+    }
+
+    .button-container {
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/common/advisory-section/advisory-section.component.scss:62]: Can this be removed if not being used?

File: client/wfnews-war/src/main/angular/src/app/components/common/advisory-section/advisory-section.component.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import { Component, Input } from '@angular/core';
+import { IconButtonArgs } from '../icon-button/icon-button.component';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/common/advisory-section/advisory-section.component.ts:5]: Can this be removed also?

File: client/wfnews-war/src/main/angular/src/app/components/common/circle-icon-button/circle-icon-button.component.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/common/circle-icon-button/circle-icon-button.component.ts:4]: Seeing this in a few places, can you let me know the use for it? from a quick search it disables the next line, I'm just curious as to the use case here. Thanks

**Comment by yzlucas** [comment]
Code looks great, just a minor change on open new tab for externalUrl. I think this is how we handled them elsewhere.

**Review by yzlucas — CHANGES_REQUESTED** [review]

File: client/wfnews-war/src/main/angular/src/app/components/full-details/cards/at-the-reception-centre-card/at-the-reception-centre-card.stories.ts [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    moduleMetadata({
+      declarations: [AtTheReceptionCentreCardComponent, ContentCardContainerComponent, IconListItemComponent, WfnewsButtonComponent]
+    })
+  ]
+};
+
+export default meta;
+type Story = StoryObj<AtTheReceptionCentreCardComponent>;
+
+export const example: Story = {
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/full-details/cards/at-the-reception-centre-card/at-the-reception-centre-card.stories.ts:21]: Does there need to be arguments here? Again, just curious

File: client/wfnews-war/src/main/angular/src/app/components/public-event-page/evac-order-details/evac-order-details.component.html [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  <two-column-content-cards-container>
+    <ng-container column-1>
+      <when-you-leave-card></when-you-leave-card>
+      <where-should-i-go-card></where-should-i-go-card>
+      <at-the-reception-centre-card></at-the-reception-centre-card>
+      <returning-home-card></returning-home-card>
+    </ng-container>
+    <ng-container column-2>
+      <connect-with-local-authorities-card></connect-with-local-authorities-card>
+      <associated-wildfire-card [incident]="incident" [isBookmarked]="isBookmarked" (bookmarkClicked)="handleBookmarkClicked($event)"
```
Comment (yzlucas) [inline client/wfnews-war/src/main/angular/src/app/components/public-event-page/evac-order-details/evac-order-details.component.html:17]: we should not need to display this empty card if no incident in available.

**Review by ssylver93 — CHANGES_REQUESTED** [review]
Looks good, just a couple of comments/queries (mostly for my benefit)

File: client/wfnews-war/src/main/angular/src/app/components/public-event-page/evac-order-details/evac-order-details.component.ts [hunk 8]
```diff
@@ -31,7 +31,7 @@ export class EvacOrderDetailsComponent {
       border: 'none'
     },
     clickHandler: () => {
-      window.open('https://www.emergencyinfobc.gov.bc.ca/');
+      window.open('https://www.emergencyinfobc.gov.bc.ca/', '_blank');
```
Comment (yzlucas) [inline client/wfnews-war/src/main/angular/src/app/components/public-event-page/evac-order-details/evac-order-details.component.ts:34]: thanks for addressing this, but might as well do the same to all other window.open (externalUrl) in this page

**Review by ssylver93 — APPROVED** [review]

**Review by yzlucas — APPROVED** [review]

**Pull request merged**